### PR TITLE
Scrollbar

### LINKED ISSUE #108: Remove the scroll bar after “Who uses EvalAI”
### Observed Behaviour 
Scroll bar is visible under `Who uses EvalAI` on normal screen. The scroll bar is used for medium and small screen. Find attached screenshot.

### Expected Behaviour
The scroll bar should not be visible on normal screen.

@RishabhJain2018 

a

### PR DESCRIPTION
**Pull request by Abhi58**
Fix #108 : After this commit, the scroll bar after “Who uses EvalAI” is removed and is only working for medium and small screens.

Changes proposed in this pull request:
Changes : 
1) Changed the overflow of x from scroll to auto in src/app/components/home/partners/partners.component.scss
2) included the max-width value for medium and small screens and set overflow-x to visible.

The screenshots of the changes are added below:
1) image

2) image

### CODE REVIEW

**Comment by Abhi58** [comment]
@RishabhJain2018 @gautamjajoo There is the problem with [surge credentials](https://travis-ci.org/Cloud-CV/EvalAI-ngx/builds/507571473#L648) as it is not able to login and thus failing the travis build. The same thing happened for PR #96 . Please look into this issue and fix it.

**Comment by RishabhJain2018** [comment]
Hey @Abhi58, Don't worry about it. I'll fix it.

File: src/app/components/privacy-policy/privacy-policy.component.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit, Inject, HostListener } from '@angular/core';
-import { DOCUMENT } from '@angular/common';
+import { DOCUMENT } from '@angular/platform-browser';
```
Comment (Shekharrajak) [inline src/app/components/privacy-policy/privacy-policy.component.ts:2]: Any reason for this change ?

File: src/app/components/faq/faq.component.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<app-header-static></app-header-static>
```
Comment (Shekharrajak) [inline src/app/components/faq/faq.component.html:1]: PR name look completely different with this changes :o 

We need to wait till surge deployed URL, to check the working copy.

**Review by Shekharrajak — COMMENTED** [review]
Screenshot written in PR description is to show UI changes not the code changes (why people will ask for code changes screenshot if they have git ? )

**Review by gautamjajoo — COMMENTED** [review]
@Abhi58 Seems that you have fixed two issues in one PR. Make different PRs for different issues. Also this is a duplicate of https://github.com/Cloud-CV/EvalAI-ngx/pull/103.

File: src/app/components/privacy-policy/privacy-policy.component.ts (hunk 1 above)
Comment (Abhi58) [reply src/app/components/privacy-policy/privacy-policy.component.ts:2]: Yes @Shekharrajak . Both the imports provide reference to the window object. But in this case `@angular/common` was not working as it was not picking the document body object. So, i have changed it to `@angular/platform-browser`.

**Comment by Abhi58** [comment]
> @Abhi58 Seems that you have fixed two issues in one PR. Make different PRs for different issues. Also this is a duplicate of #103.

Sure @gautamjajoo . Thanks for the feedback.

**Comment by RishabhJain2018** [comment]
@Abhi58 Please fix the Travis build

**Comment by Shekharrajak** [comment]
@Abhi58 , There are many ways to do. Here is the simple step to remove unrelated changes : 

1. git reset (you have to do `soft` reset since `hard` reset will remove your changes as well) to your last master commit.
2. Now all the changes are still present and you can git add relevent files and folder, then commit. 
3. To reflect new commit in this PR, you have to force push `git push -f origiin branch_name`.

If you want to be safe and don't want to mess with commit and git commands, I will recomment you to before doing this, create a new branch. Now come back to this branch `scrollbar` and try above changes.

You can easily find many articles if you stuck in any git command. Hope this will help.

**Comment by RishabhJain2018** [comment]
Closing this one since this is the duplicate of #103 .

**Pull request closed without merging**